config.xml

```diff
@@ -60,6 +60,8 @@
     <preference name="AutoHideSplashScreen" value="false" />
     <preference name="phonegap-version" value="cli-7.1.0" />
     <preference name="HeaderColor" value="#eb7125" />
+    <engine name="android" spec="^6.2.3" />
+    <plugin name="cordova-plugin-dialogs" spec="~1.3.3" />
```

Comment:
Think these needs to be downgraded to work with PhoneGap build. Same goes for package-lock.json and package.json, I think. @navro

Reply:
Yes, please use the versions specified on master :)